feedthefox/devices/views.py

```diff
@@ -1,8 +1,30 @@
 from django.shortcuts import render
 
+import django_filters
+
+from feedthefox.devices.models import Device
+
+
+class DeviceFilter(django_filters.FilterSet):
+    type = django_filters.AllValuesFilter()
+    manufacturer = django_filters.AllValuesFilter()
+
+    class Meta:
+        model = Device
+        fields = ['manufacturer', 'type']
+
+    def __init__(self, *args, **kwargs):
+        super(DeviceFilter, self).__init__(*args, **kwargs)
+
+        # Add "Any" entry to choice fields.
+        for name, f in self.filters.items():
```

Comment:
In both cases you have to add options to the filter, or not? If I have understood this correctly, the `ChoiceField` seems cleaner since you can populate the list of options once and pass it to the filter instead of adding items through the `__init__.